eng/pipelines/spec-gen-sdk-java.yml

```diff
@@ -0,0 +1,51 @@
+parameters:
+  - name: SpecRepoBranch
+    type: string
+    default: 'main'
+  - name: SpecRepoCommit
+    type: string
+    default: 'HEAD'
+  - name: SdkRepoBranch
+    type: string
+    default: 'main'
+  - name: ConfigType
+    type: string
+    values:
+      - 'TypeSpec'
+      - 'OpenAPI'
+    default: 'TypeSpec'
+    displayName: 'Configuration Type'
+  - name: TspConfigPath
+    type: string
+    default: 'specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml'
+    displayName: 'Path to TypeSpec Configuration file'
+  - name: ReadmePath
```

Comment:
I assume one of these will be ignored. Should they be just one string parameter that accepts either?

Reply:
It sounds good to combine them to one parameter from user perspective.